Add release notes for the new IA features (FAB, Me and Reader) (#14127)
wordpress-mobile/WordPress-iOS: RELEASE-NOTES.txt

14.9
-----

> * Streamlined navigation: now there are fewer and better organized tabsposting shortcuts and more, so you can find what you need fast.

Reviewer: ```suggestion
* Streamlined navigation: now there are fewer and better organized tabs, posting shortcuts and more, so you can find what you need fast.
```

> * My Site: the "Add Posts and Pages" features has been moved: there is a new "Floating Action Button" in "My Site" that lets you create a new post or page without having to navigate to another screen.

Reviewer: ```suggestion
* My Site: the "Add Posts and Pages" features has been moved. There is a new "Floating Action Button" in "My Site" that lets you create a new post or page without having to navigate to another screen.
```

> * My Site: the "Me" section has been moved: there is a new button on the top right of "My Site" that lets you access the "Me" section from there.

Reviewer: ```suggestion
* My Site: the "Me" section has been moved. There is a new button on the top right of "My Site" that lets you access the "Me" section from there.
```
